From e195d6f6a00af83f8159b09fa8539fd5c0a26e40 Mon Sep 17 00:00:00 2001
From: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
Date: Mon, 5 Aug 2019 16:32:27 +0100
Subject: [PATCH 011/322] [FIO todrop] mx6_common: Share configs to skip low
 level init

This commit replicates for i.MX6 the same fix as was applied for
i.MX7 to share and aggregate the logic for skipping low-level init.

NOTE: Should be converted to the CONFIG_SKIP_LOWLEVEL_INIT defconfig
option during updating u-boot to v2022.01.

Signed-off-by: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 include/configs/mx6_common.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/include/configs/mx6_common.h b/include/configs/mx6_common.h
index bb515f2af0b..0605dc8d06e 100644
--- a/include/configs/mx6_common.h
+++ b/include/configs/mx6_common.h
@@ -68,4 +68,13 @@
 #else
 #define TEE_ENV "tee=no\0"
 #endif
+
+/*
+ * If OPTEE_SKIP_LOWLEVEL_INIT is flagged indicate CONFIG_SKIP_LOWLEVEL_INIT and
+ * subsequently skip over reserved a range of reserved low-level bit twiddling.
+ */
+#ifdef CONFIG_OPTEE_SKIP_LOWLEVEL_INIT
+#define CONFIG_SKIP_LOWLEVEL_INIT
+#endif
+
 #endif
-- 
2.34.1

